src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -72,10 +73,10 @@
      * 3. Returning "true" will add the provided task to the list of updated tasks, if not there
      *    already.
      *
-     * @param other The other task
+     * @param that The other task
      * @return True if it can be, false if not
      */
-    boolean canRunConcurrentWith(AITask<O> other);
+    boolean canRunConcurrentWith(AITask<?> that);
```

Comment:
If its changed to the ```?``` then you put the pressure on the dev to ensure that the task you pass in is relevant to the task at hand.

At least with what is there, you will be confident that the task being passed in will have an owner that is of the same type of your task's owner (when I make it obey the contract, thanks @ST-DDT ).

With that said, thoughts?